src/options/components/navigation/NavLink.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router'
+import classNames from 'classnames'
+import styles from './styles.css'
+
+const NavLink = ({ route, currentLocation, state }) => {
+    const navClasses = classNames({
+        [styles.navLink]: true,
+        [styles.isActive]: isActive(route),
+    })
+
+    const navIcon = classNames({
+        [styles.navIcon]: true,
+        'material-icons': true,
+    })
+
+    function isActive(route) {
```

Comment:
This is another state derivation that can be derived in a container method and passed down as a prop to this view component